doc/acting/discontinued/teb/motion_planner.py

```
@@ -0,0 +1,1020 @@
+            sum_sqrt = np.dot(diff.T, diff)
+            return np.sqrt(sum_sqrt)
+
+        def map_corner(dist):
+            if dist < 8:  # lane_change
+                return 8
+            elif dist < 25:
+                return 4
+            elif dist < 50:
+                return 7
+            else:
+                8  # TODO add return
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)